### PR TITLE
add "invoking an API" content, change endpoint to covid-19, and include minor theme changes

### PR DESCRIPTION
**Pull request by shaniR**
## Purpose
Fixes #https://github.com/wso2-enterprise/choreo/issues/5395
Fixes #https://github.com/wso2-enterprise/choreo/issues/5435

### CODE REVIEW

File: en/docs/apis/create-and-publish-your-first-api.md [hunk 1]
```diff
@@ -2,38 +2,32 @@
...
     | **API Version**  | `1.0.0`                                          |
-    | **Endpoint**     | `http://www.mocky.io/v2/5185415ba171ea3a00704eed`|
+    | **Endpoint**     | `https://disease.sh`|
 
 5. Click **Create**. This displays an overview of the API.
 
     ![API overview](../assets/img/apis/api-overview.png){.cInlineImage-full}
        
 6. Under **API Configuration**, click **Resources**. Then follow this procedure to add a resource to the API:
     1. Select `GET` as the HTTP verb.
```
Comment (anupama-pathirage) [inline en/docs/apis/create-and-publish-your-first-api.md:29]: Shouldn't we bold `HTTP verb` as we are referring to UI field name here.

File: en/docs/apis/create-and-publish-your-first-api.md [hunk 2]
```diff
@@ -2,38 +2,32 @@
...
-    2. Enter `hello` as the URI pattern.
+    2. Enter `/v3/covid-19/states` as the URI pattern.
```
Comment (anupama-pathirage) [inline en/docs/apis/create-and-publish-your-first-api.md:30]: Shouldn't we highlight `URI pattern` ?

File: en/docs/apis/create-and-publish-your-first-api.md [hunk 3]
```diff
@@ -56,8 +50,9 @@ Follow this procedure to test the API revision that you deployed:
 2. Click the `GET` resource you added to expand it.
 
     ![API GET resource](../assets/img/apis/api-resource.png){.cInlineImage-full}
-
-3. Click **Try it out** and then click **Execute**. You can see `{"hello": "world"}` as the response from the API.
+
```
Comment (anupama-pathirage) [inline en/docs/apis/create-and-publish-your-first-api.md:53]: - On line 35 above (`Now you have created the API and added a GET resource to it.`): We need to highlight the word `GET`  with backticks right?
- In line 41 : `Create Revision and Deploy` should be changed to `Create a  Revision and Deploy`

File: en/docs/apis/create-and-publish-your-first-api.md [hunk 4]
```diff
@@ -56,8 +50,9 @@ Follow this procedure to test the API revision that you deployed:
 2. Click the `GET` resource you added to expand it.
 
     ![API GET resource](../assets/img/apis/api-resource.png){.cInlineImage-full}
-
-3. Click **Try it out** and then click **Execute**. You can see `{"hello": "world"}` as the response from the API.
+    
+3. Click **GET TEST KEY** to obtain an access token.
```
Comment (anupama-pathirage) [inline en/docs/apis/create-and-publish-your-first-api.md:54]: We better move this step to step number 2 as it appears at the top of the page. Then we can ask to do Click the `GET` resource you added to expand it.` as the third step.

File: en/docs/apis/create-and-publish-your-first-api.md [hunk 5]
```diff
@@ -68,9 +63,6 @@ Follow this procedure to publish the API and view the published API on the API D
 
 1. Click **Publish** in the left pane.
 2. On the **Lifecycle Management** page, click **Publish**.
-
-    ![Publish the API](../assets/img/apis/publish.png){.cInlineImage-full}
-
 3. Click **Go to Developer Portal**.
```
Comment (anupama-pathirage) [inline en/docs/apis/create-and-publish-your-first-api.md:66]: Shouldn't we mention about intermediate sign-in step here

File: en/docs/apis/create-and-publish-your-first-api.md [hunk 6]
```diff
@@ -56,8 +50,9 @@ Follow this procedure to test the API revision that you deployed:
 2. Click the `GET` resource you added to expand it.
 
     ![API GET resource](../assets/img/apis/api-resource.png){.cInlineImage-full}
-
-3. Click **Try it out** and then click **Execute**. You can see `{"hello": "world"}` as the response from the API.
+    
+3. Click **GET TEST KEY** to obtain an access token.
+3. Click **Try it out** and then click **Execute**. You can see the Covid-19 totals for all US states as the response from the API.
```
Comment (anupama-pathirage) [inline en/docs/apis/create-and-publish-your-first-api.md:55]:
```suggestion
3. Click **Try it out** and then click **Execute**. You can see the COVID-19 totals for all US states as the response from the API.
```

File: en/docs/apis/create-and-publish-your-first-api.md [hunk 7]
```diff
@@ -80,3 +72,13 @@ Follow this procedure to publish the API and view the published API on the API D
...
+
+## Step 5: Invoke the API 
+Follow this procedure to invoke the API from the API Developer Portal:
+
+1. Click **Credentials** in the left pane.
+2. On the **Credentials** page, click **GENERATE ACCESS TOKEN**. Copy the generated access token.
+3. Click **Try Out** in the left pane. 
+4. Paste the copied access token in the **Access Token** field.
+5. Click the `GET` resource you added to expand it.
+6. Click **Try it out** and then click **Execute**. You can see the Covid-19 totals for all US states as the response from the API.
```
Comment (anupama-pathirage) [inline en/docs/apis/create-and-publish-your-first-api.md:84]:
```suggestion
6. Click **Try it out** and then click **Execute**. You can see the COVID-19 totals for all US states as the response from the API.
```

File: en/docs/apis/create-and-publish-your-first-api.md (hunk 5 above)
Comment (shaniR) [reply en/docs/apis/create-and-publish-your-first-api.md:66]: it's not there now right? SSO seems to be working.

File: en/docs/apis/create-and-publish-your-first-api.md (hunk 1 above)
Comment (shaniR) [reply en/docs/apis/create-and-publish-your-first-api.md:29]: done

File: en/docs/apis/create-and-publish-your-first-api.md (hunk 2 above)
Comment (shaniR) [reply en/docs/apis/create-and-publish-your-first-api.md:30]: done

File: en/docs/apis/create-and-publish-your-first-api.md (hunk 3 above)
Comment (shaniR) [reply en/docs/apis/create-and-publish-your-first-api.md:53]: Done

File: en/docs/apis/create-and-publish-your-first-api.md (hunk 4 above)
Comment (shaniR) [reply en/docs/apis/create-and-publish-your-first-api.md:54]: done

File: en/docs/apis/create-and-publish-your-first-api.md (hunk 6 above)
Comment (shaniR) [reply en/docs/apis/create-and-publish-your-first-api.md:55]: done

File: en/docs/apis/create-and-publish-your-first-api.md (hunk 7 above)
Comment (shaniR) [reply en/docs/apis/create-and-publish-your-first-api.md:84]: done

File: en/docs/apis/create-and-publish-your-first-api.md (hunk 5 above)
Comment (anupama-pathirage) [reply en/docs/apis/create-and-publish-your-first-api.md:65]: SSO  working, I meant no need to provide credentials, But it says shows intermediate page, `Signing you in`. So user might get confused, why sign in again.

Comment (shaniR) [reply en/docs/apis/create-and-publish-your-first-api.md]: Ah ok. done

File: en/docs/apis/create-and-publish-your-first-api.md [hunk 8]
```diff
@@ -68,15 +62,22 @@ Follow this procedure to publish the API and view the published API on the API D
...
+
+## Step 5: Invoke the API 
+Follow this procedure to invoke the API from the API Developer Portal:
+
+1. Click **Credentials** in the left pane.
+2. On the **Credentials** page, click **GENERATE ACCESS TOKEN**. Copy the generated access token.
+3. Click **Try Out** in the left pane. 
+4. Paste the copied access token in the **Access Token** field.
+5. Click the `GET` resource you added to expand it.
+6. Click **Try it out** and then click **Execute**. You can see the COVID-19 totals for all US states as the response from the API.
```
Comment (anupama-pathirage) [inline en/docs/apis/create-and-publish-your-first-api.md:83]: It seems the page ended abruptly.  Consider rewording and moving the line 73 (`Congratulations!...`) to here.

Comment (shaniR) [reply en/docs/apis/create-and-publish-your-first-api.md:81]: done

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**